delphi/translators/for2py/loop_handle.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
+
+"""File: loop_handle.py
+
+Purpose: Read the Fortran AST (obtained from rectify.py) and refactor it
+         to remove the breaks and returns from within loop statements
+
+"""
+
+import sys
+import pickle
+import argparse
+import re
+import json
+import copy
+from typing import Dict
+from delphi.translators.for2py.format import list_data_type
+from delphi.translators.for2py import For2PyError, syntax
+
+
+class RefactorBreaks(object):
+    """This class defines the refactor state of the intermediate AST while
+    removing the breaks and returns inside while loops
+    """
+
+    def __init__(self):
+        self.find_opp = {
+            ".eq.": ".ne.",
```

Comment:
This table of complementary operators also appears in rectify.py.  Can we pull this out into syntax.py?

Reply:
Just put the dictionary into `syntax.py` and updated their usage here and in `rectify.py`